### PR TITLE
Translate 2.6.0-preview1 to Simplified Chinese (zh_cn)

### PR DESCRIPTION
**Pull request by dsh0416**

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
thanks for the translation, the english source has changed, please rebase on top of it.

**Comment by dsh0416** [comment]
This PR and #1753, have followed up the latest #1750 patch, and have already rebased to get a clean patch. By the time this commit followed up the translation of #1750 and rebased, the original commit still hasn't been merged into master. So this commit forked from an outdated base, but the patch has already followed up the latest post. I'm not sure if it is necessary to rebase to the latest master again.

File: zh_cn/news/_posts/2018-02-24-ruby-2-6-0-preview1-released.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+我们高兴地宣布 Ruby 2.6.0-preview1 已发布。
+
+Ruby 2.6.0-preview1 是 Ruby 2.6.0 系列的首个测试版本。此次 preview1 发布比以往早，因为其包括了重要新特性：JIT。
+
+## JIT
+
+Ruby 2.6 引入了 JIT (Just-in-time) 编译器的初始实现。
+
+JIT 编译器旨在提升任何 Ruby 程序的执行速度。不同于其他语言中常见的 JIT 编译器，Ruby 的 JIT 编译器进行 JIT 编译的过程非常独特。其将生成的 C 代码打印至磁盘并 spawn 常见的 C 编译器进行代码的生成工作。详见：[Vladimir Makarov 的 MJIT 组织分支](https://github.com/vnmakarov/ruby/tree/rtl_mjit_branch#mjit-organization)
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2018-02-24-ruby-2-6-0-preview1-released.md]: We can remove 分支 from `Vladimir Makarov 的 MJIT 组织`.

File: zh_cn/news/_posts/2018-02-24-ruby-2-6-0-preview1-released.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+如何使用：在命令行或 $RUBYOPT 环境变量中指定 "--jit" 参数。指定 "--jit-verbose=1" 将允许打印 JIT 编译过程中的调试信息。详见 "ruby --help" 查看更多其他选项。
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2018-02-24-ruby-2-6-0-preview1-released.md]:
```
$RUBYOPT -> `$RUBYOPT`

"--jit" -> `--jit`

"--jit-verbose=1" -> `--jit-verbose=1`

`ruby --help`
```

File: zh_cn/news/_posts/2018-02-24-ruby-2-6-0-preview1-released.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+我们正着手实现 JIT 编译器的内联方法，这将有助于大幅提升 Ruby 的性能。此外，我们计划增加支持的平台的数量，而下一个目标是支持 Visual Studio。
+
+请保持对 Ruby 新时代性能的关注。
+
+## 新特性
+
+* 新增 `Random.bytes`。[功能 #4938]
+
+* 新增 `Binding#source_location`。[功能 #14230]
+  此方法以一个二元组数组 `__FILE__` and `__LINE__` 的形式返回绑定的源代码路径。传统上，这可以通过执行 `eval("[__FILE__, __LINE__]", binding)` 来获得相同的数据。但我们计划改变这一行为让 `Kernel#eval` 忽略绑定的源代码路径 [漏洞 #4352]。所以，用户需要通过新加入的方法来替代之前的 `Kernel#eval`。
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2018-02-24-ruby-2-6-0-preview1-released.md]: 绑定 in `形式返回绑定的源代码路径` should be `binding`.

**Comment by dsh0416** [comment]
Problems mentioned have fixed.

**Comment by JuanitoFatas** [comment]
Thank you 👍 ! Merged as 6ee4eb8

**Pull request closed without merging**